### DISCUSSION TITLE
Has anyone been able to get the jmx exporter pods to sync and launch?

### DISCUSSION THREAD

**Discussion by SLetchSQ** [atlassian/data-center-helm-charts#723] [General]
I noticed there's config for requests/limits in the monitoring section, suggesting there's supposed to be a pod for running the jmx exporter? However even though I have no sync errors, I don't see anything trying to launch a pod for jmx exporting, or am I getting it wrong as to how it's supposed to work?

**Comment by grawert** [comment]
have you set `monitoring.exposeJmxMetrics: true`? The jmx exporter is copied and enabled only if this value is set to true.

**Reply by SLetchSQ** [reply]
One thing I'm noticing, is it doesn't seem to be paying attention to my custom jar location, if I increment version on my chart, I see nothing trying to sync down the new location

**Comment by SLetchSQ** [comment]
Yep

`monitoring:

  # -- Expose JMX metrics with jmx_exporter https://github.com/prometheus/jmx_exporter
  #
  exposeJmxMetrics: true

  # --  JMX exporter init container configuration
  #
  jmxExporterInitContainer:

    # -- Whether to run JMX exporter init container as root to copy JMX exporter binary to shared home volume.
    # Set to false if running containers as root is not allowed in the cluster.
    #
    runAsRoot: true

    # -- Custom SecurityContext for the jmx exporter init container
    #
    customSecurityContext: {}

    # -- Resources requests and limits for the JMX exporter init container
    # See: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
    #
    resources:
       requests:
         cpu: "1m"
         memory: "20Mi"
       limits:
         cpu: "1"
         memory: "1G"

  # -- Annotations added to the jmx service
  #
  jmxServiceAnnotations: {}

  # -- Fetch jmx_exporter jar from the image. If set to false make sure to manually copy the jar
  # to shared home and provide an absolute path in jmxExporterCustomJarLocation
  #
  fetchJmxExporterJar: false

  # -- Image repository with jmx_exporter jar
  #
  jmxExporterImageRepo: bitnami/jmx-exporter

  # -- Image tag to be used to pull jmxExporterImageRepo
  #
  jmxExporterImageTag: 1.0.1

  # -- Port number on which metrics will be available
  #
  jmxExporterPort: 9090

  # -- JMX exporter port type
  #
  jmxExporterPortType: ClusterIP

  # -- Location of jmx_exporter jar file if mounted from a secret or manually copied to shared home
  #
  jmxExporterCustomJarLocation: "/var/atlassian/application-data/shared-home/jmx_prometheus_javaagent-1.0.1.jar"

  # -- Custom JMX config with the rules
  #
  jmxExporterCustomConfig: {}
  #  rules:
  #   - pattern: ".*"`
